app/components/developer_profile_create_component.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class DeveloperProfileCreateComponent < ApplicationComponent
+  def initialize(user)
+    @user = user
```

Comment:
Please add a read only property instead of referencing the ivar in the view. `attr_reader :user`